src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -274,32 +277,30 @@ class PaymentInformation extends React.Component {
   }
 }
 
-const isEvssAvailable = createIsServiceAvailableSelector(
+const isEvssAvailableSelector = createIsServiceAvailableSelector(
   backendServices.EVSS_CLAIMS,
 );
 
 const mapStateToProps = state => {
   const directDepositIsSetUp = directDepositIsSetUpSelector(state);
-  const isEligible = isEvssAvailable(state);
-  const isNotBlocked = !directDepositIsBlocked(state);
+  const isEvssAvailable = isEvssAvailableSelector(state);
   const isEligibleToSignUp = directDepositAddressIsSetUp(state);
 
   return {
     directDepositIsSetUp,
-    isEligible,
+    isEvssAvailable,
```

Comment:
`isEligible` had become a very confusing name because rules for direct deposit eligibility have changed a lot since work started on this feature.